VanDerPol/FMI1ME.xml

```diff
@@ -15,19 +15,19 @@
     <ScalarVariable name="time" valueReference="0" causality="internal" variability="continuous" description="Simulation time">
       <Real/>
     </ScalarVariable>
-    <ScalarVariable name="x0" valueReference="1" causality="output" description="the first state">
+    <ScalarVariable name="x0" valueReference="1" causality="output" description="Position">
       <Real start="2" fixed="true"/>
     </ScalarVariable>
-    <ScalarVariable name="der(x0)" valueReference="2">
+    <ScalarVariable name="der(x0)" valueReference="2" description="Derivative of x0">
       <Real/>
     </ScalarVariable>
-    <ScalarVariable name="x1" valueReference="3" causality="output" description="the second state">
+    <ScalarVariable name="x1" valueReference="3" causality="output" description="Velocity">
```

Comment:
x1 is not a velocity

Reply:
If xo is the position coordinate its derivative can be seen as velocity.

Reply:
`x0` and `x1` are the first and second elements of the position vector (called `x` and `y` on Wikipedia) with the respective derivatives `der(x0)` and `der(x1)`.
The equation can also be used to model electrical circuits where `x` is not a position and `der(x)` is not a velocity.

Reply:
This is the [two-dimensional form](https://en.wikipedia.org/wiki/Van_der_Pol_oscillator#Two-dimensional_form) where `x1` = `der(x0)`. It is not a position vector.

Reply:
> This is the [two-dimensional form](https://en.wikipedia.org/wiki/Van_der_Pol_oscillator#Two-dimensional_form) where `x1` = `der(x0)`. It is not a position vector.

Any feedback on my latest comment?

With the changes of #259 it is inconsistent.